src/python/dxpy/app_builder.py

```diff
@@ -192,7 +192,7 @@ def _fix_perm_filter(tar_obj):
     return tar_obj
 
 
-def upload_resources(src_dir, project=None, folder='/', ensure_upload=False, force_symlinks=False, brief=False):
+def upload_resources(src_dir, project=None, folder='/', ensure_upload=False, force_symlinks=False, brief=False, resources_dir=None, worker_resources_subpath=""):
```

Comment:
Should the default value here be `worker_resources_subpath='.'` to match the previous behavior?

Reply:
I've played around with it and the behavior is same for paths "/", "" and ".".

This is because the default path is "/" and joining paths any of these three with anything else will result in the same behavior. I found using "." a little bit confusing - would you rather we keep it "."?